### ISSUE TITLE
position image (bug?)

### ISSUE THREAD

**Issue by login007** [newbthenewbd/grav-plugin-tinymce-editor#52]
Hello!
How can I position the text to the right of the image and at the top so that the text is indented from the image.
Thnx!

![изображение](https://user-images.githubusercontent.com/62464559/88058490-3556aa00-cb6c-11ea-9ba3-8ad0d9eb4812.png)

**Comment by thekenshow** [comment]
I don't think you can do that in TinyMCE without editing the HTML directly to add a style="float:left;" attribute.

Have a look at the Shortcode plugin, maybe the columns or figure shortcode can help.
